lexicons/app/bsky/feed/post.json

```diff
@@ -15,6 +15,11 @@
             "items": {"type": "ref", "ref": "#entity"}
           },
           "reply": {"type": "ref", "ref": "#replyRef"},
+          "media": {
```

Comment:
We should chat through this, originally (IIRC) this was what `embeds` were meant to be for

Reply:
oh it totally was. i yoloed this too quick lol